src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -773,7 +775,7 @@ namespace HVACVariableRefrigerantFlow {
               MySuppCoilPlantScanFlag(true), airLoopNum(0), isInOASys(false), isInAirLoop(false), isInZone(false), isSetPointControlled(false),
               coolSPActive(false), heatSPActive(false), coolLoadToSP(0.0), heatLoadToSP(0.0), coilTempSetPoint(0.0), suppTempSetPoint(0.0),
               controlZoneMassFlowFrac(1.0), zoneSequenceCoolingNum(0), zoneSequenceHeatingNum(0), coolCoilAirInNode(0), coolCoilAirOutNode(0),
-              heatCoilAirInNode(0), heatCoilAirOutNode(0)
+              heatCoilAirInNode(0), heatCoilAirOutNode(0), coolingCoilActive(true)
```

Comment:
The unit test probably works because this new var is initialized to true. I would set this to false here since the schedule should be checked before all other calculations where it will get set true if the schedule allows.

Reply:
This variable should be set to `true` unless certain specific conditions requires it to be `false`.  That is why I initialized to `true`.